docs/src/app/components/pages/home.jsx

```diff
@@ -68,12 +67,12 @@ const HomePage = React.createClass({
       },
     };
 
-    styles.h2 = this.mergeStyles(styles.h1, styles.h2);
+    styles.h2 = Object.assign(styles.h1, styles.h2);
 
     if (this.isDeviceSize(StyleResizable.statics.Sizes.LARGE)) {
-      styles.tagline = this.mergeStyles(styles.tagline, styles.taglineWhenLarge);
-      styles.h1 = this.mergeStyles(styles.h1, styles.h1WhenLarge);
-      styles.h2 = this.mergeStyles(styles.h2, styles.h2WhenLarge);
+      styles.tagline = Object.assign(styles.tagline, styles.taglineWhenLarge);
+      styles.h1 = Object.assign(styles.h1, styles.h1WhenLarge);
+      styles.h2 = Object.assign(styles.h2, styles.h2WhenLarge);
     }
```

Comment:
Something is mess up here:
![capture d ecran 2016-02-12 a 12 03 05](https://cloud.githubusercontent.com/assets/3165635/13005220/abd63122-d180-11e5-8b77-90c416991343.png)